laravel/resources/views/emails/user-daily-digest.blade.php

```diff
@@ -0,0 +1,15 @@
+<h1>Daily Reminder about your shifts!</h1>
+
+<p>
+    Thank you for being a part of <b>{{ env('SITE_NAME') }}</b>.
+    Here's your daily dump of some weird stuff.
+</p>
+
+@foreach($notification_metadata as $metadata)
+    @include('emails.'.$metadata['layout'], $metadata)
+    <br>
```

Comment:
I managed to manually run this and I got a wierd result, it looks like its putting the date of when it starts as the start time
`<h1>Daily Reminder about your shifts!</h1>

<p>
    Thank you for being a part of <b>Your Volunteer Database</b>.
    Here's your daily dump of some weird stuff.
</p>

    <h1>You're signed up for the Vice President Of Human Resources shift!</h1>

<p>
   This is a confirmation email for the Vice President Of Human Resources shift
   you recently picked up for the Vice President Of Human Resources event.
</p>

<p>
    This shift takes place on 2019-09-05 between the times of
    2019-09-05 and 09:00:00.
</p>

<p>
    If you did <b>NOT</b> sign-up for this shift or would like to <b>CANCEL</b> this
    shift, <a href="http://localhost:8000/slot/163/view">click here</a>.
</p>

<p>
    Otherwise, we look forward to seeing you there!
</p>
    <br>

<p>
    If any of these look strange, please <a href="http://localhost:8000">log into your account</a> and review your shifts.
</p>
`